scripts/configure-hypervisor.sh

```diff
@@ -245,6 +245,7 @@ generate_qemu_options() {
 	aarch64) ;;
 	x86_64) qemu_options+=(size:--disable-tcg) ;;
```

Comment:
/me notes that x86 should be at the bottom if we are going alphabetical, but that's not your problem or has to be part of this PR :-)